routes/index.js

```diff
@@ -58,9 +58,21 @@ router.post('/register', function(req, res, next) {
       return res.redirect('/register');
     }
 
+    if (!req.body.first_name) {
+      req.flash('error', 'Please enter your first name.')
+      return res.redirect('/register');
+    }
+
+    if (!req.body.last_name) {
+      req.flash('error', 'Please enter your last name.')
+      return res.redirect('/register');
+    }
+
```

Comment:
Won't these be taken care [below](frontend/static_src/js/templates/SiteStatusItemTemplate.ejs)